frontend-implicaction/src/app/core/services/api-endpoints.service.ts

```diff
@@ -239,6 +239,10 @@ export class ApiEndpointsService {
     return ApiEndpointsService.createUrlWithPathVariables(Uris.JOBS.BASE_URI, [JobId]);
   }
 
+  archiveJobPostingListEndpoint(): string {
```

Comment:
```suggestion
  toggleArchiveJobsEndpoint(): string {
```

 * on évite d'utiliser 'List' en général on préfère mettre un 's' pour signifier qu'il s'agit d'une collection
 * Job~~Posting~~s, on simplifie en oubliant le côté `posting`
 * et le toggle c'est mieux que archive parce qu'on chercher à 'toggler' le carcactère archivé